src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2649,3 +2649,4 @@ Note\:\ The\ study\ directory\ should\ be\ empty.=Note: The study directory shou
 Warning\:\ The\ selected\ directory\ is\ not\ empty.=Warning: The selected directory is not empty.
 Warning\:\ Failed\ to\ check\ if\ the\ directory\ is\ empty.=Warning: Failed to check if the directory is empty.
 Warning\:\ The\ selected\ directory\ is\ not\ a\ valid\ directory.=Warning: The selected directory is not a valid directory.
+Compare\ with\ duplicate\ entries=Compare with duplicate entries
```

Comment:
The files should always end with a linebreak. There are very few exceptions to that rule.